server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -741,16 +750,203 @@ def _create_tsdb_tables(
+    ) -> list[mlrun.common.schemas.model_monitoring.FunctionSummary]:
+        """
+        Retrieve a list of all the model monitoring functions with their summaries. Note that the response includes
+        both monitoring application real time functions and monitoring infrastructure functions.
+        :param start:            The start time of the statistics of the monitoring applications. Applicable
+                                 only when `include_status` is set to True. If not set, the default is 24 hours ago.
+        :param end:              The start time of the statistics of the monitoring applications. Applicable only
+                                 when `include_status` is set to True. If not set, the default is now.
+        :param names:            List of monitoring application function names to filter the response. Default is None.
+        :param labels:           List of labels to filter the response. Default is None.
+        :param include_stats:    If True, the function will include the statistics of the monitoring applications.
+                                 Currently, the statistics include the number of detections and possible detections.
+        :param include_infra:    If True, include the model monitoring infrastructure functions in the response.
+
+        :return:                A list of FunctionSummary objects, each representing a model monitoring function.
+        """
+
+        # Enrich response with infra functions
+        functino_summaries_list, base_period = self._enrich_function_summary_with_infra(
+            include_infra=include_infra
+        )
+
+        # Enrich response with monitoring applications
+        return self._enrich_function_summary_with_applications(
+            functino_summaries_list=functino_summaries_list,
+            base_period=base_period,
+            start=start,
+            end=end,
+            names=names,
+            labels=labels,
+            include_stats=include_stats,
+        )
+
+    def _enrich_function_summary_with_infra(
+        self,
+        include_infra: bool = True,
+    ) -> tuple[list[mlrun.common.schemas.model_monitoring.FunctionSummary], int]:
+        """
+        Enrich the function summaries list with the model monitoring infrastructure functions.
+        In addition, it returns the base period of the controller function.
+
+        :param include_infra: If True, include the model monitoring infrastructure functions in the response.
+
+        :return: Returns a tuple containing a list of FunctionSummary objects and the base period of the controller
+                 function. If `include_infra` is False, the list will be empty.
+
+        """
+        function_summaries_list = []
+        base_period = 0
+        if include_infra:
+            infra_mm_functions = self.list_model_monitoring_functions(
+                format_=mlrun.common.formatters.FunctionFormat.full, infra_only=True
+            )
```

Comment:
This seems to be conflicting logic - `include_infra` means to get infra+user-apps, while the list here asks for `infra_only`, so will only return infra functions.